site/src/pages/components/selectmenu.mdx

```diff
@@ -378,3 +379,12 @@ Like other form control elements, selectmenu supports validity checking attribut
 * [HTML spec for `checkValidity()`](https://html.spec.whatwg.org/#dom-cva-checkvalidity)
 * [HTML spec for `reportValidity()`](https://html.spec.whatwg.org/#dom-cva-reportvalidity)
 * [HTML spec for `setCustomValidity()`](https://html.spec.whatwg.org/#dom-cva-setcustomvalidity)
+
+## Why not improve `<select>`
+
+Instead of creating a new `<selectmenu>` element, we could improve the existing select element to meet the same needs. However, there are a few problems with this (borrowed from the [Custom Controls UI Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/ControlUICustomization/explainer.md):
+
+- Changing the parsing behavior for `<select>` to allow other element children by default would likely break a lot of old sites that depend on the current behavior.
+- The `<select>` popup currently has the capability to break outside the window of the browser or of an iframe. Sites likely depend on this behavior, but we would need to remove it for security reasons if `<select>` is allowed to contain arbitrary content. So again this would need to be opt-in to avoid breaking existing sites.
+
+In order to resolve these two issues, we could have an attribute opt-in on the select element, like `<select custom>`. However, this has its own issues as well: we would have to make a major parsing change based on the presence of an attribute, which doesn't happen for other attributes/elements. What if a `<select custom>` is parsed and an inline script child of it removes its `custom` attribute before the `<select>` has been closed?
```

Comment:
@josepharhar thank you for writing this up. I wouldn't end this with this question; I'd end with the emphatic statement that it is not backwards compatible.

Reply:
Ok, I added another paragraph to the end. What do you think?

Reply:
@josepharhar How about this as I feel like the script question throws off the flow a bit:

> - The `<select>` popup currently has the capability to break outside the window of the browser or of an iframe. Sites may depend on this behavior, but we would need to remove it for security reasons if `<select>` is allowed to contain arbitrary content. 

> Given that we would be required to introduce something new into the platform (attribute, element, etc) to opt-in to the new behavior; we opted for a new element. This comes with the added benefit that over the years most developers that want customization of controls and components will not leverage `<select>` at all; akin to how developers leverage `grid` and `flex` display types for layout instead of `<table>` or `float`. Additionally, this has the potential to simplify the implementation cost since we do not have to make substantial changes to the current `<select>` parser.

@josepharhar Thoughts? @dandclark @mfreed7 if you all want to add some more weight to the parsing changes; feel free.

Reply:
ok, i replaced the ending with your suggestion